theano/tests/test_flake8.py

```diff
@@ -269,6 +269,7 @@
     "misc/tests/test_may_share_memory.py",
     "misc/tests/test_pycuda_theano_simple.py",
     "misc/tests/test_gnumpy_utils.py",
+    "misc/tests/test_pkl_utils.py",
```

Comment:
A problem that flake8 permits import expressions only on the top of a file, but I have to check if CUDA available before the import.